Apps/WebClient/src/ClientApp/src/services/winstonLogger.ts

```diff
@@ -1,14 +1,18 @@
 import { Logger, createLogger, format, transports } from "winston";
 import { ILogger } from "@/services/interfaces";
 import { injectable } from "inversify";
+import { ExternalConfiguration } from "@/models/configData";
 
 @injectable()
 export class WinstonLogger implements ILogger {
     private logger!: Logger;
 
-    public initialize(): void {
+    public initialize(config: ExternalConfiguration): void {
         this.logger = createLogger({
-            level: "debug",
+            level:
+                config.webClient.logLevel !== undefined
+                    ? config.webClient.logLevel
+                    : "warning",
```

Comment:
Default should be Information
Is the level case sensitive?  typically the values from WebClient would be capitalized.

Reply:
nice catch @sslaws ! It is actually case sensitive. I will change it to config.webClient.logLevel.toLowerCase() to be safe.

Reply:
code updated per @sslaws's comment above